app/views/lead_providers/guidance/ecf_usage.html.erb

```diff
@@ -46,19 +48,135 @@
 
 <p class="govuk-body-m">See <a href="/lead-providers/guidance/reference#get-api-v1-participants" class="govuk-link">retrieve multiple participants</a> endpoint.</p>
 
-<h2 id="declaring-that-an-ecf-participant-has-started-their-induction" class="govuk-heading-l">Declaring that an ECF participant has started their induction</h2>
+<h2 id="make-a-declaration" class="govuk-heading-l">Making a declaration for an ECF participant</h2>
+
+<h3 class="govuk-heading-m">
```

Comment:
This bit doesn't feel like a usage scenario to me. Perhaps it belongs somewhere else.